testsuite/non_trivial_obj_def.tcc

```diff
@@ -0,0 +1,94 @@
+/** for page size defs  **/
+#include <unistd.h>
```

Comment:
This header does not exist on Windows and was previously only included when on UNIX. However, there is another problem with this, see my comment below

Reply:
```suggestion
#ifdef __unix__	
#include <unistd.h>	
#endif
```